src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days > 0 && days < 3) {
+    result = days * 40;
```

Comment:
Try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content. For example `const DAILY_RENT_COST = 40`
```suggestion
    result = days * DAILY_RENT_COST;
```